app/src/main/java/com/sdpteam/connectout/event/EventID.java

```diff
@@ -0,0 +1,12 @@
+package com.sdpteam.connectout.event;
+
+import com.sdpteam.connectout.common.EntityID;
+
+public class EventID extends EntityID {
+
+    public static final EventID UNSET = new EventID("");
+
+    public EventID(String value) {
+        super(value);
+    }
+}
```

Comment:
I would advise against this approach as the `EventID` doesn't add any additional behavior to the original `EntityID`, maybe having an UNSET attribute for the `EntityID` would be better ? what do you think ?